docs/installation/environment-variables.md

```diff
@@ -124,17 +124,15 @@ Slack
-| `MAIL_SMTP_ENCRYPTION`      | `tls`                                          | Mail encryption.                                                                                                                                                            |
-| `MAIL_SMTP_AUTH`            | `false`                                        | Whether to use SMTP Auth.                                                                                                                                                   |
-| `MAIL_USERNAME`             | **Required** (when `MAIL_SMTP_AUTH` is `true`) | The username used in SMTP Auth.                                                                                                                                             |
-| `MAIL_PASSWORD`             | **Required** (when `MAIL_SMTP_AUTH` is `true`) | The password used in SMTP Auth.                                                                                                                                             |
-| `MAIL_SMTP_DEFAULT_SENDER`  | **Required**                                   | The email used in `from` and `replyTo` fields of sent email. Note: Depending on the mailserver this might be different from SMTP Auth username.                             |
-| `MAIL_SMTP_DEFAULT_MESSAGE` | `Please do not reply to this email`            | Default message body of emails.                                                                                                                                             |
-| `MAIL_SMTP_DEBUG_LEVEL`     | `0`                                            | Debug level used in [PHPMailer](https://github.com/PHPMailer/PHPMailer/blob/master/src/SMTP.php#L116-L126). `0` No output, `4` Noisy, low-level data output, rarely needed. |
+| Environment Variable   | Default                                      | Description                                                   |
+| ---------------------- | -------------------------------------------- | ------------------------------------------------------------- |
+| `MAIL_SMTP_HOST`       | `smtp.mailgun.org`                           | Mail server hostname. For example `smtp.mailserver.com`.      |
+| `MAIL_SMTP_PORT`       | `587`                                        | Mail server port.                                             |
+| `MAIL_SMTP_ENCRYPTION` | `tls`                                        | Mail encryption.                                              |
+| `MAIL_SMTP_USERNAME`   | **Required** (when authentication is needed) | The username used in SMTP authentication.                     |
+| `MAIL_SMTP_PASSWORD`   | **Required** (when authentication is needed) | The password used in SMTP authentication.                     |
```

Comment:
Maybe we can remove the SMPT ones from here, and instead just link to Laravel documentation: https://laravel.com/docs/12.x/mail#driver-prerequisites ?